### PR TITLE
Modified the null placeholder in phonenumber field to +1 in both lightMode and darkMode pages

### PR DESCRIPTION
**Pull request by FourAndHalf**
Added default country code to the previously empty box
Fixes issue #6437

**Changed the values assigned as default from ' ' to '+1' on both pages** 
 
```jsx
  const [formData, setFormData] = useState({
    companyName: '',
    role: '',
    companySize: '',
```
```diff
- phoneNumber: ' ',
+ phoneNumber: '+1',
```
```jsx
   )};
```

The place for correction that was shared before in the earlier screenshot was actually on the page's form titles
```jsx
const FORM_TITLES = [
    `Where do you work ${userDetails?.name}?`,
    'What best describes your role?',
    'What is the size of your company?',
    'Enter your phone number',
    'Enter your phone number', //dummy for styling
  ];
```

I also looked up how to change the displayed flag, but it was set using a dependency library. I don't anything can be done about it by me. 
```jsx
import PhoneInput from 'react-phone-input-2';
import 'react-phone-input-2/lib/style.css';
```

### CODE REVIEW

File: frontend/src/OnBoardingForm/OnBoardingForm.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ function OnBoardingForm({ userDetails = {}, token = '', organizationToken = '',
     companyName: '',
     role: '',
     companySize: '',
-    phoneNumber: '',
+    phoneNumber: '+1',
```
Comment (iSatVeerSingh) [inline frontend/src/OnBoardingForm/OnBoardingForm.jsx:25]: I tried it and this is not working. You should set a default county as "US" in phone input.

File: frontend/src/OnBoardingForm/OnbboardingFromSH.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ function OnbboardingFromSH({ darkMode }) {
     email: '',
     password: '',
     workspace: '',
-    phoneNumber: '',
+    phoneNumber: '+1',
```
Comment (iSatVeerSingh) [inline frontend/src/OnBoardingForm/OnbboardingFromSH.jsx:31]: Also for this.

**Comment by iSatVeerSingh** [comment]
Also update the line "Fixes issue https://github.com/ToolJet/ToolJet/issues/6437" to "Fixes #6437". So the issue can automatically close when PR get merge.

**Comment by johnsoncherian** [comment]
@FourAndHalf this solution does not seem to be working. Try passing country directly in input field as mentioned in previous comments. Refer this documentation for better understanding:
https://www.npmjs.com/package/react-phone-input-2

**Review by johnsoncherian — CHANGES_REQUESTED** [review]
@FourAndHalf this solution does not seem to be working. Try passing country directly in input field as mentioned in previous comments. Refer this documentation for better understanding:
https://www.npmjs.com/package/react-phone-input-2

**Pull request closed without merging**